ide/che-core-ide-stacks/src/main/resources/stacks.json

```diff
@@ -2403,7 +2403,11 @@
                   "protocol": "tcp"
                 }
               },
-              "volumes": {},
+              "volumes": {
+                "databaseData": {
+                  "path": "/var/lib/pgsql/data"
```

Comment:
I get an error `Volume name 'databaseData' in machine 'database' is invalid` but no issues with `databasedata`